src/hooks/useHttpClient.tsx

```diff
@@ -12,12 +14,18 @@ export const defaultAxiosInstance = axios.create({
   headers: defaultHeaders,
 });
 
+export const defaultAPIClientInstance = new APIClient<RestAPIEndpoints>(
+  defaultAxiosInstance,
+);
+
 export interface HttpClient {
   httpClient: AxiosInstance;
+  apiClient: APIClient<RestAPIEndpoints>;
```

Comment:
I think consolidating these two clients should happen but I figured I better wait to do that in a different PR.